app/models/contact_info.rb

```diff
@@ -4,16 +4,16 @@ class ContactInfo < ApplicationRecord
   before_create :set_confirmation_pin_code
   before_destroy :check_if_last_verified
 
-  validates :user, presence: true
   validates :type, presence: true
-  validates :value,
+  validates :value, # rubocop: disable Rails/UniqueValidationWithoutIndex
```

Comment:
@Dantemss can you expand on how I would do this?

Create a migration and add an index on just the [:value]?

Reply:
It may be better to put this in a separate PR.

It really depends on what we want to prevent here. The first thing to do is decide when this validation should fail: What happens if the user tries to add the same email to 2 different accounts? We could fail immediately so they can't even add 2 of the same unverified email, in which case the unique index should be something like `add_index :contact_infos, 'lower(value), type', unique: true`. Or we could fail only if they try to verify both emails, in which case it's something like: `add_index :contact_infos, 'lower(value), type', unique: true, where: 'verified'`

And then the model validation can be changed to match the index which should get rid of the rubocop warning.